x-pack/plugins/maps/public/classes/layers/solution_layers/observability/create_layer_descriptor.ts

```diff
@@ -39,7 +39,7 @@ import { getDefaultDynamicProperties } from '../../../styles/vector/vector_style
 
 // redefining APM constant to avoid making maps app depend on APM plugin
 export const APM_INDEX_PATTERN_ID = 'apm_static_index_pattern_id';
-export const APM_INDEX_PATTERN_TITLE = 'apm-*';
+export const APM_INDEX_PATTERN_TITLE = 'traces-apm*,logs-apm*,metrics-apm*,apm-*';
```

Comment:
I think this use of APM_INDEX_PATTERN_TITLE is going to need to be updated with this change, https://github.com/elastic/kibana/blob/master/x-pack/plugins/maps/public/classes/layers/solution_layers/security/create_layer_descriptors.ts#L38

Reply:
Correct, I have another PR up for that: https://github.com/elastic/kibana/pull/105334

Reply:
I've updated the test with the new labels based on the index pattern title. Is there any other the new pattern should be handled?